app/src/main/java/com/naxtlevelofandroiddevelopment/marvelgallery/presentation/common/PresenterBaseActivity.kt

```diff
@@ -0,0 +1,13 @@
+package com.naxtlevelofandroiddevelopment.marvelgallery.presentation.common
+
+import android.support.v7.app.AppCompatActivity
+
+abstract class PresenterBaseActivity : AppCompatActivity() {
+
+    abstract val presenter: Presenter
+
+    override fun onDestroy() {
```

Comment:
if you make presenter lifecycle-aware, you can react to something like onViewDestroyed()